components/coachmark/stories/template.js

```diff
@@ -129,3 +130,25 @@ export const Template = (args, context) => {
 		</div>
 	`;
 };
+
+/* Displays open and closed action menus in a single story. */
+export const CoachmarkMenuStatesTemplate = (args, context) => Container({
+	withBorder: false,
+	withHeading: false,
+	wrapperStyles: {
+		columnGap: "100px",
+		rowGap: "200px",
+	},
+	content: html`
+		${Container({
+			withBorder: false,
+			heading: "With action menu",
+			content: Template({...args, isOpen: true}, context),
+		})}
+		${Container({
+			withBorder: false,
+			heading: "Without action menu",
+			content: Template({...args, hasActionMenu: false}, context),
+		})}
+	`
```

Comment:
This is perfect but just so you see a possible alternative, you can also pass content in using an array and then you don't have to add the html wrapper here:

```suggestion
	content: [
		Container({
			withBorder: false,
			heading: "With action menu",
			content: Template({...args, isOpen: true}, context),
		}),
		Container({
			withBorder: false,
			heading: "Without action menu",
			content: Template({...args, hasActionMenu: false}, context),
		})
	]
```